### DISCUSSION TITLE
[Multi-panel Antenna Arrays]

### DISCUSSION THREAD

**Discussion by Advaith2212** [NVlabs/sionna#313] [Q&A]
I was wondering if there is an option to configure multi-panel antenna array configuration.

The current method is to configure a single panel antenna is using the `PlanarArray` object. 

`scene.tx_array = PlanarArray(1,1,0.5,0.5,"tr38901", polarization="V")`

Can you help me with this

**Comment by jhoydis** [comment]
Hi,

Multi-panel arrays are not implemented. However, you can freely control the positions of the array's antenna elements via `scene.tx_array.positions` ([link](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.AntennaArray.positions)).  So it should be pretty straight-forward to create any desired array architecture.

Hope that helps.
